docs/src/main/paradox/docs/delta/api/resources-api.md

```diff
@@ -34,6 +34,28 @@ The json payload for create and update operations cannot contain keys beginning
 
 @@@
 
+## Nexus metadata
+
+When using the endpoints described on this page, the responses can contain the following metadata:
+
+- `_self`: unique address of this resource in the current Nexus Delta deployment
+    - Each resource has an `@id`, which is only unique within a project. The `_self` in contrast to the `@id` will
+      uniquely identify a resource across all projects in Nexus Delta.
+    - A `GET` request can be performed on a `_self` to obtain the underlying resource
+- `_incoming`: identifier leading to a list of resources which make references to the current resource
+- `_outgoing`: identifier leading to a list of resources which are referred to by the current resource
```

Comment:
I think you would just say "a lot of resources", or is there a single identifier leading to a list?

Reply:
You have to follow/query the URI to get to the list; I will try to improve it

Reply:
Simplified and pointing to link